app/controllers/support/providers_controller.rb

```diff
@@ -48,8 +63,24 @@ def provider
       @provider ||= Provider.find(params[:id])
     end
 
-    def provider_params
+    def update_provider_params
       params.require(:provider).permit(:provider_name)
     end
+
+    def create_provider_params
+      params.require(:provider).permit(:provider_name,
+                                       :provider_code,
+                                       :recruitment_cycle_id,
+                                       :email,
+                                       :telephone, sites_attributes: %i[id
```

Comment:
Since this is a new site/organisation being created, do we need the id here?

Reply:
Good point, removed the ids.